app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,30 @@
+class CommentsController < ApplicationController
+
+  def index
+    @comment = Comment.find(params[:id])
+  end
+
+  def show
+    @comment = Comment.find(params[:id])
+  end
+
+  def new
+    @job = Job.find(params[:job_id])
+    @comment = Comment.new()
+  end
+
+  def create
+    @job = Job.find(params[:job_id])
+    @comment = @job.comments.new(comment_params)
+    @comment.save
```

Comment:
what happens if the comment doesn't save due to invalid data? how does a user know that their comment hasn't been saved?